internal/command/tokens/list.go

```diff
@@ -44,8 +47,49 @@ func runList(ctx context.Context) (err error) {
 	apiClient := flyutil.ClientFromContext(ctx)
 	out := iostreams.FromContext(ctx).Out
 	var rows [][]string
+	var scope = "app"
+
 
-	scope := flag.GetString(ctx, "scope")
+	// Get Organization Name, if org flag passed 
+	orgFlag := flag.GetString(ctx, "org")
+	var org *fly.Organization
+	if orgFlag!=""{
+		// Get the org name if flag indicated
+		org, err = orgs.OrgFromEnvVarOrFirstArgOrSelect(ctx)
+		if err != nil {
+			return fmt.Errorf("failed retrieving org %w", err)
+		}
+		scope = "org"
+	}
+	
+	// Get App Name, if no org flag, OR if app flag passed
+	// Make sure app belongs to a selected org, else return error
+	appFlag := flag.GetString(ctx, "app")
+	var appName string
+	if orgFlag == "" || appFlag!="" {
+		
+		appName = appconfig.NameFromContext(ctx)
```

Comment:
this reads like it will not work the way I'd assume it to. Let's say I'm in `project-a` and I do `fly tokens list --app project-b`, I think this will still list stuff from `project-a`, no? I think the `appFlag` already gives you the name of the `appName`

I could be wrong here, but without testing this and just looking at code, I feel the above would happen - feel free to resolve if you tested and it is not the case :)

Reply:
Hi @fliepeltje ! Thanks for looking into this, I appreciate it! 

Ok!  

-> You're right about flag.GetString(ctx, "app") returning the argument for the app flag.
-> On the other hand, I tested the behavior you've described( when the command `tokens list --app  project-b` is run in a Fly initialized directory with a different app, say `project-a` ). The resulting value returned from `appconfig.NameFromContext(ctx)` is still actually `project-b`, see.

Current Directory fly app: `new-app-sprinfrost...`
Flag value passed: `--app base11`
Command run: `bin/flyctl.exe tokens list --app base11`
Result: 
```
```